templates/default/mesos_slave.yaml.erb

```diff
@@ -0,0 +1,13 @@
+# Generated by Chef, local modifications will be overwritten
```

Comment:
ok, thanks for the clarification! This is 👍 to me right now.

Just so you know, in a future major version of this cookbook we'll move towards generalizing the installation of integrations (to simplify the cookbook and make it more flexible) so we'll probably stop supporting integration-specific yaml templates.